app/services/taxonomy/bulk_publish_taxon.rb

```diff
@@ -1,5 +1,5 @@
 module Taxonomy
-  class BulkUpdateTaxon
+  class BulkPublishTaxon
```

Comment:
I'm guessing the why here is that you want to add another different service with this name?